client/src/js/agreement/CreateAgreementContainer.jsx

```diff
@@ -31,14 +31,14 @@ function CreateAgreementContainer () {
         <Grid item xs={12} md={6} className={classes.cell}>
           <Link to={`/sign/${AgreementType.INDIVIDUAL}`}>
             <Button variant='contained' color='primary' size='large'>
-              Individual Contributions
+              Only Yourself
```

Comment:
Bill asked to change `Only yourself` and `Your Employer` to `Individual/Corporate contributions`

> change the button “Yourself” to “Individual Contributions”
> change the button “Your employer” to “Corporate Contributions”

Reply:
How about
```suggestion
              Only Yourself / Sign Individual CLA
```

Reply:
works for me